doc/proposal.md

```diff
@@ -1,11 +1,11 @@
 
-##1. This webiste is a collectible themed website. It has images of professionally graded sports cards, non-sports trading cards, comic books, coins, and much more. The user can enter search words that will search eBay for a list of "sold listings" for that particular professionally graded items, over a certain time interval. The first graph displayed shows the sold listings for that item over a time interval. Three graphs will be displayed in a seqeuntial Stack sequence, after the sold listing graph. The first graph will predict future prices for the collectible using an "Out of sample"/step forward algorithm. The second graph will display predicted future prices for the collectible using a Multivariate Linear regression model. The third graph will display future price predictions for the collectible using a Deep Neural Network.
+### 1. This website is a collectible themed website. It has images of professionally graded sports cards, non-sports trading cards, comic books, coins, and much more. The user can enter search words that will search eBay for a list of "sold listings" for that particular professionally graded items, over a certain time interval. The first graph displayed shows the sold listings for that item over a time interval. Three graphs will be displayed in a seqeuntial Stack sequence, after the sold listing graph. The first graph will predict future prices for the collectible using an "Out of sample"/step forward algorithm. The second graph will display predicted future prices for the collectible using a Multivariate Linear regression model. The third graph will display future price predictions for the collectible using a Deep Neural Network. <h3>
 
-##2. This webiste is targeted to customers that invest in high-end, expensive, professionally graded collectibles. To help them predict future prices, to know when to buy and sell.
+### 2. This website is targeted to customers that invest in high-end, expensive, professionally graded collectibles. To help them predict future prices, to know when to buy and sell. <h3>
 
-##3. This is unlike any website known on the web, to the best of our knowledge.
+### 3. This is unlike any website known on the web, to the best of our knowledge. <h3>
 
-##4. This webiste is worth buying because of the amount of money that goes into professionally graded collectibles. Example: 1996 "Sports Illustrated for kids" Tiger Woods, graded BGS 9.5 sold for over $100,000 in 2001. Now, it's worth nothing near what it use to be. Making investments in such things as CGC graded "silver age" X-men #1, graded 9.6 Mint condition.
+### 4. This webiste is worth buying because of the amount of money that goes into professionally graded collectibles. Example: 1996 "Sports Illustrated for kids" Tiger Woods, graded BGS 9.5 sold for over $100,000 in 2001. Now, it's worth nothing near what it use to be. Making investments in such things as CGC graded "silver age" X-men #1, graded 9.6 Mint condition. <h3>
```

Comment:
```suggestion
### 4. This website is worth buying because of the amount of money that goes into professionally graded collectibles. Example: 1996 "Sports Illustrated for kids" Tiger Woods, graded BGS 9.5 sold for over $100,000 in 2001. Now, it's worth nothing near what it use to be. Making investments in such things as CGC graded "silver age" X-men #1, graded 9.6 Mint condition. <h3>
```